mongodb-advanced/components/index.js

```diff
@@ -24,9 +24,7 @@ module.directive('exFocus', FocusDirective);
 module.factory('exPeopleService', PeopleService);
 
 /* @ngInject */
-module.config($routeProvider => {
-  $routeProvider.when('/', {
-    title: 'Basic-modals Example',
-    template: '<br-people></br-people>'
-  });
-});
+module.config($routeProvider => $routeProvider.when('/', {
```

Comment:
Although this is fine syntactically, I think it makes it more difficult to add additional routes if someone were to want to extend this example.